frontend/jest.config.ts

```diff
@@ -1,4 +1,7 @@
 import type { Config } from '@jest/types';
+import dotenv from 'dotenv';
+
+dotenv.config({ path: '.env.test' });
```

Comment:
Same, can we read this from the `.evn` file rather than creating a new one i.e. `.evn.test`.